src/panels/lovelace/cards/hui-humidifier-card.ts

```diff
@@ -26,6 +26,39 @@ import { HumidifierCardConfig } from "./types";
 
 @customElement("hui-humidifier-card")
 export class HuiHumidifierCard extends LitElement implements LovelaceCard {
+  private _resizeObserver?: ResizeObserver;
```

Comment:
Can we use the `ResizeController` from `lit-labs` instead?

Reply:
I tried without success but now I succeeded 😅